packages/playground-server/src/node.ts

```diff
@@ -146,7 +145,10 @@ export default class NodeWrapper {
       {
         STORE_KEY_PREFIX: "store"
       },
-      provider || ethers.getDefaultProvider(networkOrNetworkContext as string),
+      provider ||
+        new JsonRpcProvider(
+          `https://${networkOrNetworkContext}.infura.io/metamask`
```

Comment:
I don't know when we ended up with this `networkOrNetworkContext` thing. It looks weird.